apps/designer/app/designer/features/style-panel/shared/style-info.ts

```diff
@@ -0,0 +1,37 @@
+import type { Style, StyleProperty, StyleValue } from "@webstudio-is/css-data";
+import { useMemo } from "react";
+
+export type StyleValueInfo = {
+  value: StyleValue;
+  local?: StyleValue;
+};
+
+export type StyleInfo = {
+  [property in StyleProperty]?: StyleValueInfo;
+};
+
+export const useStyleInfo = ({
+  localStyle,
+  browserStyle,
+}: {
+  localStyle: Style;
+  browserStyle: Style;
+}) => {
+  const styleInfoData = useMemo(() => {
+    const styleInfoData: StyleInfo = {};
+    for (const [property, value] of Object.entries(browserStyle)) {
+      styleInfoData[property as StyleProperty] = {
+        value,
+      };
+    }
+    for (const [property, value] of Object.entries(localStyle)) {
```

Comment:
we figured it out: this is a temporarily overriding value, we will start using a combination of local and default (normal/auto etc) as a value, lets add this as a comment